proto/config/appconfig.proto

```diff
@@ -210,6 +210,9 @@ message AppInstanceConfig {
   // allow AppInstance to discover other AppInstances
   // attached to its network instances
   bool allow_to_discover = 24;
+
+  // This edge-node UUID for the Designate Node for the Application
+  string designated_node_id = 25;
```

Comment:
FWIW I still think this will cause technical debt since the controller has to be involved in re-selecting a designated node if the previously selected one fails. So approving this holding my nose.